components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/ApiResource.java

```diff
@@ -113,9 +113,11 @@ private void populateApiList(MessageContext messageContext) {
         for (API api: apis) {
 
             JSONObject apiObject = new JSONObject();
-
-            String apiUrl = serverUrl.equals("err") ? api.getContext() : serverUrl + api.getContext();
-
+            String versionUrl = "";
+            if (!api.getVersion().isEmpty()) {
+                versionUrl = "/" + api.getVersion();
+            }
+            String apiUrl = serverUrl.equals("err") ? api.getContext() : serverUrl + api.getContext() + versionUrl;
```

Comment:
We need to do the same in, https://github.com/wso2/micro-integrator/blob/380949eaf9592fc6f9fafb9ce89130d12378bdb9/components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/ApiResource.java#L166?

Reply:
Fixed